### PR TITLE
Remove erroneous content from user guide documentation.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Remove forgotten unfinished documentation from user guide and update scanning instructions.

#### Rationale:

- better accuracy in oscap scanning parameters
- Remove Nonsense doc.

### CODE REVIEW

File: docs/manual/user_guide.adoc [hunk 1]
```diff
@@ -131,10 +131,10 @@ Putting these arguments together, a properly formatted command would be:
 
 ----
 $ sudo oscap xccdf eval --profile stig-rhel6-server \
---results /root/ssg-results.xml \
---report /root/ssg-report.xml \
---cpe /usr/share/scap/ssg/ssg-rhel6-cpe-dictionary.xml \
-/usr/share/scap/ssg/ssg-rhel6-xccdf.xml
+--results /tmp/results.xml \
+--report /tmp/report.xml \
```
Comment (yuumasato) [inline docs/manual/user_guide.adoc]: It is better if a report has `html` extension.

File: docs/manual/user_guide.adoc [hunk 2]
```diff
@@ -163,11 +163,7 @@ Result  pass
 
 === Result Interpretation
 
-HTML Results
```
Comment (yuumasato) [inline docs/manual/user_guide.adoc]: You can just suggest to open the `html` file in favorite browser.

File: docs/manual/user_guide.adoc (hunk 1 above)
Comment (ggbecker) [reply docs/manual/user_guide.adoc]: Agreed.

File: docs/manual/user_guide.adoc (hunk 2 above)
Comment (ggbecker) [reply docs/manual/user_guide.adoc]: Ok. So I'll put back the sections and write this recommendation.

**Comment by yuumasato** [comment]
Looks good, thanks!

**Pull request merged**